libexec/scoop-update.ps1

```diff
@@ -57,6 +57,22 @@ if(($PSVersionTable.PSVersion.Major) -lt 5) {
 }
 $show_update_log = get_config SHOW_UPDATE_LOG $true
 
+# Migrate PATH EnvVar to SCOOP_PATH
+$movedPath = Remove-Path -Path "$scoopdir\apps\*" -Quiet -PassThru
```

Comment:
Will this be run every time on `scoop update`?

Reply:
Yes, check every time since we don't know if the user has mixed PATH. But could be removed in next version.

Reply:
I see, makes sense.